neo4j-examples/paradise-papers-django
# Feature-41: sphinx-quickstart setup, docs initial files.

Closes [Ticket](https://trello.com/c/Vm5dzvM8/41-as-a-developer-i-want-to-install-sphinx-so-that-i-can-create-a-tutorial-in-github-pages)

As a developer I want to install sphinx so that I can create a tutorial in github pages
Changes:
 - Created initial docs files through the command `sphinx-quickstart`
 - Add table of contents.


## Testing

Steps to manually verify the change:
1. _Setup_: Change directory to` docs/`
2. _Exercise_: Run `make html`, open the sub-directory `_build/` and open generated html docs(`index.html`)
3. _Verify_: Initial tutotial Index of content should be there.
